src/oomd/util/Fs.cpp

```diff
@@ -461,12 +461,12 @@ std::unordered_map<std::string, int64_t> Fs::getVmstat(
 std::unordered_map<std::string, int64_t> Fs::getMeminfo(
     const std::string& path) {
   char name[256] = {0};
-  unsigned long val;
+  unsigned long long val;
   std::unordered_map<std::string, int64_t> map;
 
   auto lines = readFileByLine(path);
   for (auto& line : lines) {
-    int ret = sscanf(line.c_str(), "%255[^:]:%*[ \t]%lu%*s\n", name, &val);
+    int ret = sscanf(line.c_str(), "%255[^:]:%*[ \t]%llu%*s\n", name, &val);
```

Comment:
And this would accordingly use the SCNu64 macro instead of hardcoding the long long type.